css/evaluations.css

```diff
@@ -1,50 +1,42 @@
-body{
-	background-color: white;
+.mainTable {
+    width: 60%;
+    margin: 100px auto;
 }
 
-.footer{
-	margin-bottom: 0;
+.mainTableSkin {
+    font-size: 20px;
+    border-collapse: collapse;
 }
 
-.mainTableClass{
-	width: 60%;
-	margin: 0 auto;
-	margin-top: 100px;
-	margin-bottom: 100px;
-
-	font-size: 20px;
-	border-collapse: collapse;
+.mainTable th {
+    text-align: left;
 }
 
-.mainTableClass th{
-	text-align: left;
+.mainTable tr {
+    height: 50px;
+    border-bottom: 1px solid black;
 }
 
-.mainTableClass tr{
-	height: 50px;
-
-	border-bottom: 1px solid black;
+.mainTable th,
```

Comment:
Just making sure that you indeed want to be targeting only `th` elements inside a `.mainTable` ancestor, but also targeting _all_ `td` elements, regardless of whether or not they have a `.mainTable` ancestor.